command/harness/dlite/gitspaces_cleanup.go

```diff
@@ -0,0 +1,67 @@
+package dlite
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"github.com/drone-runners/drone-runner-aws/command/harness"
+	"github.com/sirupsen/logrus"
+	"github.com/wings-software/dlite/client"
+	"github.com/wings-software/dlite/httphelper"
+)
+
+type GitspacesVMCleanupTask struct {
+	c *dliteCommand
```

Comment:
Can we make the name a bit more descriptive so that it is easier to understand in the method body?
such as `cmd` or `dlcmd`